app/models/course/registration.rb

```diff
@@ -0,0 +1,29 @@
+class Course::Registration < ActiveRecord::Base
+  stampable
+
+  scope :staff, -> { where(role: roles.values_at(:teaching_assistant, :manager, :owner)) }
```

Comment:
can you try to find a way such that we can expand this staff keyword to this array? Maybe augment the CourseUser::Roles enum. not sure if it's possible.